cfme/tests/cli/test_appliance_update.py

```diff
@@ -215,6 +219,11 @@ def test_update_distributed_webui(ext_appliances_with_providers, appliance, requ
     wait_for(do_appliance_versions_match, func_args=(appliance, ext_appliances_with_providers[1]),
              num_sec=900, delay=20, handle_exception=True,
              message='Waiting for appliance to update')
+    ext_appliances_with_providers[0].evmserverd.wait_for_running()
```

Comment:
Question: Why not loop over `ext_appliances_with_providers`?

Reply:
Well I don't think that it is worthy and more readable to do with only 2 appliances. It is 2 lines vs 2 lines deal. But whatever...